Shield.Communication/Services/ServiceBase.cs

```diff
@@ -234,7 +234,8 @@ internal bool InstrumentSocket(IInputStream input, IOutputStream output)
                     uint val = this.dataReader.ReadByte();
                     if (val < 255)
                     {
-                        Debug.Write(val + ",");
+                        //Debug.Write(val + ",");
+                        Debug.Write((char)val);
```

Comment:
no, this outputs each character so that something like "Hello" is written together rather than "H e l l o"